content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -247,9 +249,23 @@ message ContainerDevices {
 }
 ```
 
+### `GetAllocatableResources` gRPC endpoint {#grpc-endpoint-getallocatableresources}
+
+{{< feature-state state="beta" for_k8s_version="v1.23" >}}
+
 GetAllocatableResources provides information on resources initially available on the worker node.
 It provides more information than kubelet exports to APIServer.
 
+{{< note >}}
+`GetAllocatableResources` should only be used to evaluate [allocatable](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable)
+resources on a node. If the goal is to evaluate free/unallocated resources it should be used in
+conjunction with the List() endpoint. The result obtained by `GetAllocatableResources` would remain
+the same unless the underlying resources exposed to kubelet change. This happens rarely but when
+it does (for example: CPUs taken online or offline; devices added or removed), client is expected to call
```

Comment:
this is correct for device changes: changes to device availability (hotplug/hotunplug, device health changes) should be reflected to the allocatability as reported by the endpoint; for cpu and memory calling again the endpoint is not sufficient, the kubelet must be restarted (!!). This is a more general kubelet limitation, but still.

I'd just rephrase the sentence between parens ().

Reply:
Good point, will fix this.